### PR TITLE
Updated Velocity to latest

### PR DESCRIPTION
**Pull request by egg82**
Kyori text was the only thing needing updating.

I don't know why you're using tabs in the pom.xml, but I tried to keep it as best as possible. Intellij doesn't like them much.

### CODE REVIEW

**Comment by egg82** [comment]
Heads-up, Velocity officially releases its first version today (woo!) - would be good to have this updated and working for the release.

**Comment by js6pak** [comment]
@aikar can you merge this?

File: velocity/src/main/java/co/aikar/commands/VelocityMessageFormatter.java [hunk 1]
```diff
@@ -12,6 +12,6 @@ public VelocityMessageFormatter(TextColor... colors) {
     @Override
     @SuppressWarnings("deprecation")
     String format(TextColor color, String message) {
-        return ComponentSerializers.LEGACY.serialize(ComponentSerializers.LEGACY.deserialize(message).color(color));
+        return LegacyComponentSerializer.legacy().serialize(LegacyComponentSerializer.legacy().deserialize(message).color(color));
```
Comment (js6pak) [inline velocity/src/main/java/co/aikar/commands/VelocityMessageFormatter.java]: Shouldn't you use `LegacyComponentSerializer.INSTANCE` like in [kyori.text sample](https://github.com/KyoriPowered/text#serializing-and-deserializing-components)?

Comment (astei) [reply velocity/src/main/java/co/aikar/commands/VelocityMessageFormatter.java]: `LegacyComponentSerializer.INSTANCE` was deprecated with `LegacyComponentSerializer.legacyLinking()` becoming an option (which makes `LegacyComponentSerializer` act more like the BungeeCord chat API and CraftBukkit).

**Comment by js6pak** [comment]
When this will be available on repo?

**Comment by chickeneer** [comment]
@js6pak I have deployed it for you. Sorry for the delay

**Pull request merged**